dynamics-nav/SETSTAMP-Function--File-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETSTAMP Function (File)"
+description: SETSTAMP Function (File) Sets a timestamp for a file.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.